content/what-is-rbac/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: What is RBAC? 
+date: "2022-10-20"
+description: "Learn about RBAC and advantages + disadvantages compared to ABAC."
+cover: "what-is-rbac-cover.png"
+category: "programming"
+author: "SuperTokens Team"
+---
+
+
```

Comment:
Need to add something in the middle of the title and the sub heading cause it renders in a weird way where you see the title - "What is RBAC?" and then right below, you see a sub heading "What is Roles-Based Access Control?"